pandas/io/clipboards.py

```diff
@@ -9,7 +9,7 @@
 
 def read_clipboard(sep=r'\s+', **kwargs):  # pragma: no cover
     r"""
-    Read text from clipboard and pass to read_table. See read_table for the
+    Read text from clipboard and pass to read_csv. See read_csv for the
```

Comment:
Hmm...this is arguably breaking (the `sep` default is different for both functions), but since we explicitly have a default arg for `sep`, this *probably* should be okay (for at least the vast majority of users).

Reply:
Same, maybe I am missing something, but I think the only difference between `read_table` and `read_csv` is anyway being overridden here, thus making the change non-breaking.

Reply:
Same as above.  It should probably be fine.  Don't worry about it.